prow/apis/prowjobs/v1/types.go

```diff
@@ -522,6 +522,16 @@ type DecorationConfig struct {
 	// set the same as this request. Could be overridden by memory request
 	// defined explicitly on prowjob.
 	DefaultMemoryRequest *resource.Quantity `json:"default_memory_request,omitempty"`
+
+	// PodPendingTimeout is after how long the controller will perform a garbage
```

Comment:
```suggestion
	// PodPendingTimeout defines how long the controller will wait to perform garbage
```

Following comments should also be updated in a similar manner

Reply:
I just wanted to stay consistent, with the [existing comment](https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L628) in the [config.go](https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go), I didn't want to have two different comments on the same option, but I can change it

Reply:
Ah I see. It is just improper grammar. Go ahead and fix the other comment too.